orangecontrib/network/widgets/OWNxTwitterGraph.py

```diff
@@ -0,0 +1,248 @@
+import twitter
+import networkx as nx
+from Orange.widgets import widget, gui
+from Orange.data import Domain, StringVariable, Table
+import numpy as np
+from Orange.widgets.credentials import CredentialManager
+import orangecontrib.network as network
+from orangecontrib.text import twitter as txt_twitter
```

Comment:
It's not common we do this. Add-ons are unreliable, at best. In particular, I don't think they make any API guarantees. If the sole reason for this is twitter API key validation, I assume, for lack of proper documentation, that the twitter lib should fail on [OAuth handshake](https://github.com/biolab/orange3-network/pull/22/files#diff-60fbdb7d6991bfefabacf79cc5c22f0dR112) if the access token were incorrect?